scripts/pvscheck.sh

```diff
@@ -373,7 +373,12 @@ run_analysis() {(
     analyze \
       --lic-file PVS-Studio.lic \
       --threads "$(get_jobs_num)" \
+      --exclude-path src/cjson \
+      --exclude-path src/mpack \
       --exclude-path src/xdiff \
+      --exclude-path src/nvim/regexp.c \
```

Comment:
was there a discussion around this? even though the code is "owned" by vim we forward issues to them on a regular basis.

Reply:
OK.